data/cameras.xml

```diff
@@ -8530,8 +8530,8 @@
 			<ColorRow y="4">GGBGGR</ColorRow>
 			<ColorRow y="5">GGRGGB</ColorRow>
 		</CFA2>
-		<Crop x="0" y="0" width="-126" height="0"/>
-		<Sensor black="1024" white="16383"/>
+		<Crop x="0" y="0" width="-128" height="0"/>
```

Comment:
> Narrow crop by 2 pixels to make output image 6032x4032, to match RawImageWidth as found in that EXIF.

No point in doing that, as per https://github.com/darktable-org/rawspeed/blob/develop/README.md

```
RawSpeed does NOT…
* crop the image to the same sizes as manufactures, but supplies biggest possible images.
```